File: postman/sdks/Specs SDKs/Orders/cli/sdk/orders/patch_order_request_status.go
package orders

type PatchOrderRequestStatus string

const (
	PatchOrderRequestStatusPending    PatchOrderRequestStatus = "pending"
	PatchOrderRequestStatusProcessing PatchOrderRequestStatus = "processing"
	PatchOrderRequestStatusShipped    PatchOrderRequestStatus = "shipped"
	PatchOrderRequestStatusCancelled  PatchOrderRequestStatus = "cancelled"
)

File: postman/sdks/Specs SDKs/Orders/typescript/src/services/orders/models/patch-order-request-status.ts
import { z } from 'zod';

export const patchOrderRequestStatus = z.union([
  z.literal('pending'),
  z.literal('processing'),
  z.literal('shipped'),
  z.literal('cancelled'),
]);

export type PatchOrderRequestStatus = z.infer<typeof patchOrderRequestStatus>;
